.github/workflows/ci.yml

```diff
@@ -105,8 +106,7 @@ jobs:
 
     - name: Build other cross compilations
       if: contains(matrix.platform.target, 'android') == false && matrix.platform.cross == true
-      run: cargo build --workspace --exclude ipfs-http --target ${{ matrix.platform.target }}
-      # exclude http on other cross compilation targets because openssl
+      run: cargo build --workspace --target ${{ matrix.platform.target }}
```

Comment:
[Instead of commenting out the IOS builds](https://github.com/ipfs-rust/rust-ipfs/pull/128/files#r404651653) I would prefer if this line would just restored to what it was earlier:

```
run: cargo build --workspace --exclude ipfs-http --target ${{ matrix.platform.target }}
```

This would mean that IOS builds are still without `ipfs-http` if the openssl is still problematic after adding the `openssl_probe`. However if the `openssl_probe` now allows building  ipfs-http on android, the line 104 could have this `--exclude ipfs-http` removed.